packages/account/src/Sections/Verification/ProofOfOwnership/proof-of-ownership-form.jsx

```diff
@@ -0,0 +1,251 @@
+                            documentType: DOCUMENT_TYPE.proof_of_ownership,
+                            proof_of_ownership: {
+                                details: {
+                                    email: client_email,
+                                    payment_identifier: payment_method_details.payment_method_identifier,
+                                },
+                                id: payment_method_details.id,
+                            },
+                        });
+                        if (typeof processed_files === 'string') {
+                            // eslint-disable-next-line no-console
+                            console.warn(processed_files);
+                        }
+                        processed_files.forEach(async (processed_file, sub_index) => {
+                            const response = await uploader.upload(processed_file);
+                            if (response.warning) {
+                                if (response.warning.trim() === 'DuplicateUpload') {
+                                    form_ref.current.errors.data = document_upload_errors;
+                                    const { data: form_errors } = form_ref?.current?.errors;
+                                    if (typeof form_errors[card_key] !== 'object') {
+                                        form_errors[card_key] = {};
+                                    }
+                                    form_errors[card_key][card_item_key] =
+                                        form_errors?.[card_key]?.[card_item_key] ?? {};
+                                    form_errors[card_key][card_item_key].files =
+                                        form_errors?.[card_key]?.[card_item_key]?.files ?? [];
+                                    form_errors[card_key][card_item_key].files[sub_index] = localize(response.message); // Document already uploaded
+                                    setDocumentUploadErrors(form_errors);
+                                    form_ref.current.setErrors(form_errors);
+                                    form_ref.current.validateForm();
+                                    setFormState({ ...form_state, ...{ is_btn_loading: false } });
+                                } else {
+                                    setFormState({ ...form_state, ...{ is_btn_loading: false } });
+                                    // eslint-disable-next-line no-console
+                                    console.warn(response);
+                                }
+                            } else {
+                                updateAccountStatus();
+                                refreshNotifications();
+                            }
+                        });
+                    }
+                });
+            });
+        } catch (err) {
+            setFormState({ ...form_state, ...{ is_btn_loading: false } });
+            // eslint-disable-next-line no-console
+            console.warn(err);
```

Comment:
why do we need to throw error in the console?

Reply:
I'll remove it @ali-hosseini-fs

Reply:
thanks